<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPrefBPMDlg</class>
 <widget class="QWidget" name="DlgPrefBPMDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BPM Detection Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>BPM Detection</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="chkEnableBpmDetection">
        <property name="text">
         <string>Enable BPM Detection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line1">
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkDetectOnImport">
        <property name="text">
         <string>Detect Tracks BPM on Import</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkWriteID3">
        <property name="text">
         <string>Write BPM to ID3 Tag</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkAboveRange">
        <property name="toolTip">
         <string>If BPM can be detected but not within specified range</string>
        </property>
        <property name="text">
         <string>Allow BPM above the range</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpBpmSchemes">
     <property name="minimumSize">
      <size>
       <width>409</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>BPM Schemes</string>
     </property>
     <layout class="QGridLayout">
      <item row="4" column="1">
       <spacer>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" rowspan="5">
       <widget class="QListWidget" name="lstSchemes">
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::NoDragDrop</enum>
        </property>
        <property name="modelColumn">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnEdit">
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnDelete">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btnDefault">
        <property name="text">
         <string>Default</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
